### ISSUE TITLE
numpy dependencie problems

### ISSUE THREAD

**Issue by vmindel** [pachterlab/gget#157]
### What happened?

Upon searching info for my gene of interset, I got a syntax error for the numpy nan call.

The error:

`gget info ENSG00000283039 

21:00:53 - WARNING - UniProt server request for ID 'ENSG00000283039' return following error:
`np.NaN` was removed in the NumPy 2.0 release. Use `np.nan` instead.`

### gget version

0.28.6

### Operating System (OS)

macOS

### User interface

Command-line

### Are you using a computer with an Apple M1 chip?

M1

### What is the exact command that was run?

```shell
gget info
```

### Which output/error did you get?

```shell
21:00:53 - WARNING - UniProt server request for ID 'ENSG00000283039' return following error:
`np.NaN` was removed in the NumPy 2.0 release. Use `np.nan` instead.`
```

**Comment by lauraluebbert** [comment]
Hi Vova, thank you for reaching out. NumPy >= 2.0 does not support np.NaN anymore which is used in some places inside the gget module. We will fix this issue in the next gget release. In the meantime, you can install the unreleased version of gget from the dev branch:
```
pip install git+https://github.com/pachterlab/gget.git@dev
``` 

Please let me know if that fixes the issue.

**Comment by vmindel** [comment]
Thank you

**Comment by lauraluebbert** [comment]
Fixed in gget version >= 0.29.0
